common/schema-model/pom.xml

```diff
@@ -10,4 +10,12 @@
     <artifactId>smallrye-graphql-schema-model</artifactId>
     <name>SmallRye: GraphQL Common :: Schema Model</name>
     <description>A model that represents the schema</description>
+
+     <dependencies>
+        <!-- The API -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>smallrye-graphql-api</artifactId>
+        </dependency>
+    </dependencies>
```

Comment:
I had to add this in order to access Federation directives interface classes. If this is not okay, we can just hardcode the class names.